pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -226,6 +226,10 @@ type CmdProbeInputs struct {
 	// The source where we have to run the command
 	// It can be a image or inline(inside experiment itself)
 	Source string `json:"source,omitempty"`
+	// HostNetwork define the hostNetwork of the external pod
+	// It is applicable for sources other than inline
+	// it supports boolean values and default value is false
+	HostNetwork bool `json:"hostNetwork,omitempty"`
```

Comment:
Do you think we should add schema within source?

Reply:
added